src/PortToDocs/src/app/PortToDocs.cs

```diff
@@ -13,6 +13,8 @@ public static void Main(string[] args)
             ToDocsPorter porter = new(config);
             porter.CollectFiles();
             porter.Start();
+            porter.SaveToDisk();
+            porter.PrintSummary();
```

Comment:
This is what I had to take out from `Start` so that we could test using just strings. I renamed `Save` to `SaveToDisk`.